source/guides/models/repositories.md

```diff
@@ -95,6 +95,7 @@ class BookRepository < Hanami::Repository
       .where(author_id: author.id)
       .order(:published_at)
       .limit(limit)
+      .to_a
   end
 end
```

Comment:
Regarding your first remark, it is not clear as the code snippet is hypothetical anyways. I do not feel strongly about that though so if you think it's better to add `.to_a` there as well then I'll do it.

Regarding your second remark, we need to materialize the query in order to be consistent with the above remark ("The caller has an intimate knowledge of the internal mechanisms of the Repository."). If we do not call `#to_a`, then the method returns some ROM object instead of a collection of entities.